netam/common.py

```diff
@@ -67,6 +67,10 @@ def clamp_probability(x: Tensor) -> Tensor:
     return torch.clamp(x, min=SMALL_PROB, max=(1.0 - SMALL_PROB))
 
 
+def clamp_probability_above(x: Tensor) -> Tensor:
```

Comment:
Let's make this function name concordant with the function below. They are both above, right?